src/_sass/tilt.scss

```diff
@@ -116,8 +119,22 @@ $on-docs-mobile:    $header-width + $spacing-unit;
 
 @mixin relative-font-size($ratio) {
   font-size: $body-font-size * $ratio;
+@mixin mobileAndBelow {
```

Comment:
i don't think this will parse? you need to close the relative-font-size mixin before you start a new one

Reply:
OOPS — this was an obvious error that was caused by a hasty merge. Then I missed it because I was looking in the wrong branch. THAT'S why I couldn't see the error in the CSS even though it was super obvious and actually readily caught by Tilt.